### DISCUSSION TITLE
Candlestick issues with timeline and a mysterious date appearance! ooooo

### DISCUSSION THREAD

**Discussion by craigfarrell** [tradingview/lightweight-charts#712] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/431365/111876288-df493900-897c-11eb-88cd-91605ecf344a.png)
A few issues I'm running into.  
1) Where is it getting the bar on the 20th?
2) Why is it not separating the bars on the 19th?

**Comment by craigfarrell** [comment]
I should also mention that I chopped off the code under that:
candleSeries.setData(data);
chart.timeScale().fitContent();

**Comment by timocov** [comment, marked as answer]
See https://github.com/tradingview/lightweight-charts/blob/master/docs/time.md#time.

Long story short - you need to use unix timestamps if you'd like to show intraday data.
